doc/TESTING.md

```diff
@@ -278,14 +278,14 @@ Use the CMake macro `add_plugintest` for adding these tests.
 ### C++ Unit Tests
 
 C++ Unit tests are done using the Google Test framework.
-See [architectural decision](/doc/decisions/unit_testing.md).
+See [architectural decision](/doc/decisions/5_implemented/unit_testing.md).
```

Comment:
This is a consequence of the separate folders. On the one hand, I like that you're always aware of the current status of a decision. On the other hand, updating the status of a decision now means fixing all files that link to the decision.

If we want to avoid this and also avoid the merge conflicts from TOC file, we probably need to store the decision status in the decision file itself. This could be some sort of frontmatter, but also just be like this:

```markdown
# Title of the decision

> **Status**: Draft

Other content
```

Reply:
> updating the status of a decision now means fixing all files that link to the decision

Do you have a regex to do this? (Or some script?) For #4501 etc. we also need to move many documentation files...

Reply:
> > updating the status of a decision now means fixing all files that link to the decision
> 
> Do you have a script to do this?

No, but it should just be a simple regex replace (doesn't even need regex). Search for the old path, replace with the new one. Also some editors may support auto-updating links when a file is moved. I know VSCode does it sometimes, but not always.

> For https://github.com/ElektraInitiative/libelektra/issues/4501 etc. we also need to move many documentation files...

Good point. But it should be a one time thing, so I don't think we need to have the script in the repo. Whoever does the PR should just use whatever they feel like.